test/configuration/specify-configuration-path/expected-logs

```diff
@@ -0,0 +1,14 @@
+=> Capturing the files in /data/app
```

Comment:
Where in this test does it actually specify this?

Reply:
The test specifies the configuration path, not the context. All tests have the same context path, I changed this for all tests.

Reply:
OK, fair. So why does this PR add an entire set of logs for this test, but only change one line for the others?

Reply:
configuration path = path to plz.config.json

Reply:
Oh, that file was uploaded inadvertently. I couldn't write that test with the current test infra. There's no test at all

Reply:
Good catch :P